automatic/kubernetes-kompose/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,9 @@
+$ErrorActionPreference = 'Stop'
+
+$installLocation = "$(Get-ToolsLocation)\kubernetes-kompose"
+
+Write-Host "Removing Kompose from '$installLocation'"
+Remove-Item $installLocation -Recurse -Force -ea 0
+
+$newPath = $Env:Path.Replace(";$installLocation\bin", '')
+if ($newPath -eq $Env:PATH) { return }
```

Comment:
This line can be removed, it doesn't do anything at all, since it's already the last line in the script.

Reply:
Ok, no problem; I didn't do the scripts from scratch, this one is a copy/paste from git.portable

Reply:
hmm, must be an old version of git.portable then, I don't see it in our source anymore.

Reply:
yes, one from > 1mo